### PR TITLE
Added YAML/JSON/XML converter action

### PR DESCRIPTION
**Pull request by fabasoad**
Added a link to the "YAML/JSON/XML Converter action" to _Utility_ section:
- [Repository](https://github.com/fabasoad/yaml-json-xml-converter-action)
- [Marketplace](https://github.com/marketplace/actions/yaml-json-xml-converter-action)

This action allows you to take file that in _YAML/JSON/XML_ format and produces it in output in _YAML/JSON/XML_ format.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -180,6 +180,7 @@ Set up your GitHub Actions workflow with a specific version of your programming
 - [Apply templates with Jinja2](https://github.com/cuchi/jinja2-action) - Use the Jinja2 template engine to generate files from templates.
 - [Has Changes](https://github.com/UnicornGlobal/has-changes-action) - Check if there are code changes from previous steps.
 - [Mind Your Language Action](https://github.com/tailaiw/mind-your-language-action) - Detect offensive comments in issues and pull requests, and warn senders.
+- [YAML/JSON/XML Converter](https://github.com/fabasoad/yaml-json-xml-converter-action) - Takes file that in YAML/JSON/XML format and produces it in output in YAML/JSON/XML format.
```
Comment (jessicalostinspace) [inline README.md]: I would change this appended description to something more succinct:
"Converts YAML/JSON/XML file formats interchangeably. "

Comment (fabasoad) [reply README.md]: Done. Please check.

Comment (fabasoad) [reply README.md]: Not sure about the reason of Lint failures... It says that 185 links in this README file are dead but they are not. Sounds like Lint is broken. Could you please take a look?

Comment (jessicalostinspace) [reply README.md]: Looks good. Please pull master and push up again. That lint check is temporarily disabled until a more robust solution is in place.

**Pull request merged**